snowflake.json

```diff
@@ -11,11 +11,11 @@
     "product_version_regex": ".*",
     "publisher": "Splunk",
     "license": "Copyright (c) 2023 Splunk Inc.",
-    "app_version": "1.1.0",
+    "app_version": "1.1.1",
```

Comment:
Per what Tony wrote above - if we don't need to change this - why do it?

Reply:
Change rollbacked